src/CMakeLists.txt

```diff
@@ -255,19 +255,26 @@ ELSE()
 ENDIF()
 
 # Optional external dependency: zlib
-FIND_PACKAGE(ZLIB)
-IF (ZLIB_FOUND)
-	LIST(APPEND LIBGIT2_INCLUDES ${ZLIB_INCLUDE_DIRS})
-	LIST(APPEND LIBGIT2_LIBS ${ZLIB_LIBRARIES})
-	IF(APPLE OR CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
-		LIST(APPEND LIBGIT2_LIBS "z")
-		LIST(APPEND LIBGIT2_PC_LIBS "-lz")
+IF (NOT USE_BUNDLED_ZLIB)
```

Comment:
Not a fault of yours, as we have just decided to enforce a consistent CMake style: please do not put any spaces between `IF(CONDITION)` instructions